app/assets/javascripts/angular/resources/shared_graph_behavior.js

```diff
@@ -1,6 +1,6 @@
 angular.module("Prometheus.services").factory("SharedGraphBehavior", ["$http", "$timeout", "$location", "URLConfigDecoder", "URLVariablesDecoder", "ThemeManager", function($http, $timeout, $location, URLConfigDecoder, URLVariablesDecoder, ThemeManager) {
-  function commonSetup($scope) {
-    $scope.globalConfig = dashboardData.globalConfig || {
```

Comment:
this was squashing any settings made to `$scope.globalConfig` before executing `commonSetup`. so, now the config object is explicitly passed in. this file and many others would benefit from refactoring.